lighthouse-cli/test/smokehouse/readme.md

```diff
@@ -0,0 +1,121 @@
+# Smokehouse
+
+Smokehouse is the Lighthouse end-to-end/smoke test runner. It takes in a set of URLs (usually pointing to custom-built test sites), runs Lighthouse on them, and compares the results against a set of expectations.
+
+By default this is done using the Lighthouse CLI (to exercise the full pipeline) with the tests listed in [`smokehouse/test-definitions/core-tests.js`](./test-definitions/core-tests.js).
+
+## Options
+
+See [`SmokehouseOptions`](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-cli/test/smokehouse/smokehouse.js#L23).
+
+## Test definitions
+
+| Name | Type | Description |
```

Comment:
gotta run prettier on these tables man, it's oh so good ;)

Reply:
ping :)

Reply:
> ping :)

well I tried it and it seemed to help on some tables and not others so...who knows :)